dragonchain/lib/matchmaking.py

```diff
@@ -329,14 +329,18 @@ def make_matchmaking_request(
             return make_matchmaking_request(http_verb=http_verb, path=path, json_content=json_content, retry=False, authenticated=authenticated)
         elif retry and response.status_code == 403 and authenticated:
             _log.warning(
-                "[MATCHMAKING] received 403 from matchmaking. Registration is expired or dragon net config is invalid. Re-registering and trying again"
+                "[MATCHMAKING] received 403 from matchmaking. "
```

Comment:
not sure why you changed this log

Reply:
It was passing the linter (I guess we ignore PEP rules on line length?), but I'm biased against output strings I can't read without having a 30" wide monitor >_>  I can change it back if you want.

Reply:
The formatter enforces line length. It's also hard to tell that these lines are necessarily connected when reading line-seperated logs.

Reply:
That formatting doesn't actually break up the lines in the log, only in the source code.  I'll just revert it, not a hill I care about dying on. :P